### PR TITLE
Tahoe: Enable Credentials service

### PR DESCRIPTION
**Pull request by OmarIthawi**
Enabling the credentials service inside the edxapp servers.

 - Jira: [**RED-919:** Setup credentials service on Tahoe staging](https://appsembler.atlassian.net/browse/RED-919)

 * https://credentials.staging-tahoe.appsembler.com/

### CODE REVIEW

File: playbooks/amc.yml [hunk 1]
```diff
@@ -40,6 +40,12 @@
     EDXAPP_LMS_NGINX_PORT: '80'
     # Set to false if deployed behind another proxy/load balancer.
     NGINX_SET_X_FORWARDED_HEADERS: True
+    TAHOE_CREDENTIALS_ENABLE: false
+    TAHOE_NGINX_SITES:
```
Comment (OmarIthawi) [inline playbooks/amc.yml]: Try to move this to `common` playbook.

**Comment by OmarIthawi** [comment]
@thraxil Black Team wants [credentials](https://github.com/edx/credentials) to be installed for Chef. 

Could you please take a quick look and see if what we're doing aligns with our next efforts of moving services out and possibly containrizing services.

**Review by thraxil — APPROVED** [review]

File: playbooks/roles/credentials/defaults/main.yml [hunk 2]
```diff
@@ -132,7 +135,19 @@ CREDENTIALS_CORS_ORIGIN_WHITELIST_DEFAULT:
 CREDENTIALS_CORS_ORIGIN_WHITELIST_EXTRA: []
 CREDENTIALS_CORS_ORIGIN_WHITELIST: '{{ CREDENTIALS_CORS_ORIGIN_WHITELIST_DEFAULT + CREDENTIALS_CORS_ORIGIN_WHITELIST_EXTRA }}'
 
+CREDENTIALS_GIT_PROTOCOL: '{{ COMMON_GIT_PROTOCOL }}'
+CREDENTIALS_GIT_MIRROR: '{{ COMMON_GIT_MIRROR }}'
+CREDENTIALS_GIT_PATH: '{{ COMMON_GIT_PATH }}'
 CREDENTIALS_VERSION: 'master'
+CREDENTIALS_GIT_IDENTITY: null
+CREDENTIALS_REPOS:
```
Comment (bryanlandia) [inline playbooks/roles/credentials/defaults/main.yml:143]: I don't understand why this had disappeared between Ginkgo and Hawthorn Configuration branches

**Review by bryanlandia — APPROVED** [review]
Looks good.  Looks like default configuration was a little sloppy (missing some vars) for Credentials service, and you added them.

File: playbooks/roles/credentials/defaults/main.yml (hunk 2 above)
Comment (OmarIthawi) [reply playbooks/roles/credentials/defaults/main.yml:143]: Yup strange. I had to dig it up, it turns out to be part of a big refactor. Those values makes no sense for edX. So they've removed it. We want it though:

 - https://github.com/edx/configuration/commit/b6d45554f05fedd8d331fa9d8515095f68c20aeb#diff-240da08108c4d3cce14b2abd689b924eL13

**Pull request merged**